openlibrary/plugins/openlibrary/code.py

```diff
@@ -28,6 +28,7 @@
 
 from openlibrary.core.vendors import create_edition_from_amazon_metadata
 from openlibrary.utils.isbn import isbn_13_to_isbn_10, isbn_10_to_isbn_13
+from openlibrary.core.models import Edition
```

Comment:
The linting is erroring on this line:

```
openlibrary/plugins/openlibrary/code.py:31:1: E402 module level import not at top of file
1     E402 module level import not at top of fil
```

Not sure what the fix is; do you by any chance know, @cclauss ?

Reply:
https://github.com/internetarchive/openlibrary/blob/master/openlibrary/plugins/openlibrary/code.py#L20-L21 is code that gets run before all the imports are complete.  If you believe this code needs to run then just add the linter directive `# noqa: E402` to the end of the line.

Reply:
Silenced the error on this line for now; I don't know if we can move infogami features further down :/